optuna/storages/_journal/storage.py

```diff
@@ -71,10 +71,17 @@ class JournalStorage(BaseStorage):
 
         .. code::
 
+            import optuna
+
+
+            def objective(trial):
+                ...
+
+
             storage = JournalStorage(JournalFileStorage("./log_file"))
```

Comment:
Should we import JournalStorage and JournalFileStorage here?

Reply:
I added the prefix like this.
https://github.com/optuna/optuna/pull/3854/commits/90a59b31766a0650f8055004e2a101c311111eeb
Which do you think is better, with `import optuna.storages.JournalStorage` and  `import optuna.storages.JournalFileStorage` or without them?